Lib/mimetypes.py

```diff
@@ -544,17 +553,21 @@ def _default_mime_types():
         '.ass'    : 'audio/aac',
         '.au'     : 'audio/basic',
         '.snd'    : 'audio/basic',
+        '.flac'   : 'audio/flac',
         '.mka'    : 'audio/matroska',
+        '.m4a'    : 'audio/mp4',
         '.mp3'    : 'audio/mpeg',
         '.mp2'    : 'audio/mpeg',
+        '.ogg'    : 'audio/ogg',
         '.opus'   : 'audio/opus',
         '.aif'    : 'audio/x-aiff',
         '.aifc'   : 'audio/x-aiff',
         '.aiff'   : 'audio/x-aiff',
         '.ra'     : 'audio/x-pn-realaudio',
-        '.wav'    : 'audio/x-wav',
```

Comment:
This will make `mimetypes.MimeTypes().guess_extension('audio/x-wav')` no longer work. Is that expected?

Reply:
Yes, we can only have a single type per extension with this dict implementation (there's also https://github.com/python/cpython/pull/122632 that proposes dealing with duplicates; I've not looked closely at it). 

If we can only choose one, I think handling for the standards-based type is more useful than an `x-` extension.

Also `mimetypes.MimeTypes().guess_type('filename.wav')` now gives the preferred result.

This is also a reason why this kind of change is only made as a feature with no backports. But happy to revert this one if you prefer.